inst/sql/modified_update/target_pop_routine.sql

```diff
@@ -16,7 +16,7 @@ SELECT demographic_statistic.demographic_source,
 -- This will need expanding if we need further detail but this will
 -- hopefully do for now (and will make the queries a little easier to
 -- deal with)
-   AND age_from IN (0, 1, 9)
+   AND age_from IN (0, 2, 9)
```

Comment:
This change seems surprising.  Do we really not need `age_from` of 1 anymore?

Reply:
Yes. I can confirm this. We have routine age_from = 1 for MCV2. It has been changed to 2 following discussions between Tini and modellers.

Reply:
Right, but we still have lots of entries in `coverage` that have `age_from` of 1 (see `SELECT * from coverage WHERE age_from = 1 LIMIT 100`).  I think that this change will mean that we can't do a modified update for these cases any further.

Reply:
Yes. You are right. Consider only routine, RCV2 and MCV2 are still labelled age_from = 1 for all older touchstones. Then we should update them all.

Reply:
Or run this as `AND age_from IN (0, 1, 2, 9)` - changing old data seems like a poor decision

Reply:
:+1:  Done.